src/core/shared.lua

```diff
@@ -30,5 +30,36 @@ AddEventHandler("ncs_core:trace", function(message, logType)
     _NCS:trace(message, logType)
 end)
 
+local registredEvents = {}
+local function isEventRegistred(eventName)
+    for k,v in pairs(registredEvents) do
+        if v == eventName then return true end
+    end
+    return false
+end
```

Comment:
```suggestion
local registredEvents = {}

local function isEventRegistred(eventName)
    for k, v in pairs(registredEvents) do
        if (v == eventName) then 
            return (true) 
        end
    end
    return (false)
end
```
For the next PR, try to respect the code conventions present in the other files, thanks :p